product_docs/docs/postgres_distributed_for_kubernetes/1/rel_notes/1_0_rel_notes.mdx

```diff
@@ -1,28 +1,36 @@
 ---
-title: 'EDB Postgres Distributed for Kubernetes 1.0 release notes'
-navTitle: "Version 1.0"
+title: 'EDB Postgres Distributed for Kubernetes 1.0.0 release notes'
+navTitle: "Version 1.0.0"
 ---
 
-**Release date:** 15 February 2024
-
-This is the first major stable release of EDB Postgres Distributed for
-Kubernetes, a Kubernetes operator to deploy and manage
-EDB Postgres Distributed clusters.
-
-The operator implements the `PGDGroup` custom resource
-in the API group `pgd.k8s.enterprisedb.io`.
-This resource can be used to create and manage EDB Postgres Distributed clusters
-inside Kubernetes with capabilities including:
-
--   Deployment of EDB Postgres Distributed clusters with versions 5 and later
--   Additional self-healing capability on top of that of Postgres Distributed,
-    such as recovery and restart of failed PGD nodes
--   Definition of the services to connect applications to the
-    write leader of each PGD group
--   Implementation of Raft subgroups
--   Support for Local Persistent Volumes with PVC templates
--   Reuse of Persistent Volumes storage in Pods
--   TLS connections and client certificate authentication
--   Continuous backup to an S3 compatible object store
--   Pause and resume a PGD cluster, saving computational resources by temporarily
-    removing database pods while keeping the database PVCs.
+Released: 24 Apr 2024
+
+This is the first major stable release of EDB Postgres Distributed for Kubernetes, a Kubernetes operator to deploy 
+and manage EDB Postgres Distributed clusters.
+
+## Highlights of EDB Postgres Distributed for Kubernetes 1.0.0
+
+The operator implements the `PGDGroup` custom resource in the API group `pgd.k8s.enterprisedb.io`. You can use this resource 
+to create and manage EDB Postgres Distributed clusters inside Kubernetes with capabilities including:
+
+* Deployment of EDB Postgres Distributed clusters with versions 5 and later.
+* Additional self-healing capability on top of that of Postgres Distributed, such as recovery and restart of failed PGD nodes.
+* Definition of the services to connect applications to the write leader of each PGD group.
+
+!!! Note
+The EDB Postgres Distributed for Kubernetes operator leverages 
+[EDB Postgres for Kubernetes](https://www.enterprisedb.com/docs/postgres_for_kubernetes/latest/) (PG4K) and inherits many 
+of that project's capabilities. EDB Postgres Distributed for Kubernetes version 1.0.0 is based, specifically, on release 1.22 of PG4K. 
+Please refer to the [PG4K release notes](https://www.enterprisedb.com/docs/postgres_for_kubernetes/latest/rel_notes/) for more details.
+!!!
+
+## Features
+
+| Component | Description                                                                                  |
+|-----------|----------------------------------------------------------------------------------------------|
+| PGD4K     | Deployment of EDB Postgres Distributed clusters with versions 5 and later inside Kubernetes. |
```

Comment:
@jsilvela not sure if to call the component pgd4k, does it make sense here? 

For comparison, see https://www.enterprisedb.com/docs/pgd/latest/rel_notes/pgd_5.4.0_rel_notes/ where we divide the enhancements per component.

Reply:
Hmm, not against it, but we (CNP dev) don't divide by component. So possibly every update would have PGD4K, which is not wrong...

Reply:
I think I have been calling it PG4K-PGD, because I have seen it like that in other instances. So it would be good to keep it that way for consistency.

On the other side, PGD4K is shorter and maybe a better alternative for customer-facing docs.